packages/devextreme/js/viz/series/points/bar_point.js

```diff
@@ -273,16 +273,17 @@ export default _extend({}, symbolPoint, {
         that[argAxis] = arg = arg === null ? arg : arg + (that[argAxis + 'Correction'] || 0);
         that['v' + valAxis] = val;
         that['v' + argAxis] = arg + that[argIntervalName] / 2;
+        that[valIntervalName] = _abs(val - minVal);
```

Comment:
Current `that[valIntervalName]` and `newVal` values need for calculate visibility only and then the are recalculated. Let's move this calculations in the separate function not to clutter current method and renamed them. For example:
`
_checkVisibility: function(val, minVal, rotated) {
        const size = _abs(val - minVal);
        const startRealValue = Math.min(val, minVal);

        if(rotated) {
            that._calculateVisibility(startRealValue, arg, size, that.height);
        } else {
            that._calculateVisibility(arg, startRealValue, that.width, size);
        }
}
`

Reply:
done